worlds/no_roof_small_warehouse.world

```diff
@@ -0,0 +1,244 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
This file was copied from small_warehouse.world file in the same folder. The roof_b_01 model is commented out, and a gui,camera tags are added at the bottom of the file

Reply:
Sucks that .world files are static. And the documented embedded ruby script is clunky. :(